File: wireless/devices/usage/instance-get-example-2/instance-get-example-2.java
// Download the **Next-Gen** twilio-java library from:
// twilio.com/docs/libraries/java#installation-nextgen
import com.twilio.sdk.Twilio;
import com.twilio.sdk.resource.preview.wireless.device.Usage;

public class Example {

  // Find your Account Sid and Token at twilio.com/console
  public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
  public static final String AUTH_TOKEN = "your_auth_token";

  public static void main(String[] args) {
    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

    Usage usage = Usage.fetch("524116518656369")
      .setStart("2016-05-01")
      .setEnd("2016-05-31")
      .execute();
    System.out.println(usage.getPeriod().get("start"));
    System.out.println(usage.getPeriod().get("end"));
    System.out.println(usage.getDataCosts().get("total"));
  }
}

File: wireless/devices/usage/instance-get-example-2/instance-get-example-2.php
<?php
// Download the **Next-Gen** twilio-php library from:
// twilio.com/docs/libraries/php#installation-nextgen

// Include dependencies from Composer's autoload (including Twilio library)
include 'vendor/autoload.php';

$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";  // Your Account SID from www.twilio.com/console
$token = "your_auth_token"; // Your Auth Token from www.twilio.com/console

$client = new Twilio\Rest\Client($sid, $token);

$usage = $client->preview->wireless->devices(
  "524116518656369")->usage()->fetch(array(
    'start' => "2016-05-01",
	'end' => "2016-05-01"
  ));

echo $usage->period['start'], "\n";
echo $usage->period['end'], "\n";
echo $usage->dataCosts['total'], "\n";